Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Name the node stage "builder"
+FROM node:14 AS builder
+# Set working directory
+WORKDIR /devhub
+# Copy all files from current directory to working dir in image
+COPY package.json ./
+# install node modules
+RUN npm install
+RUN npm install -g gatsby-cli
```

Comment:
Curious what do we need the CLI for? Could we CMD [npm, run, develop, -h, 0.0.0.0]?

Reply:
yup.Gatsby develop is erroring out, if I dont install gatsby cli